assets/theme/colors.js

```diff
@@ -0,0 +1,20 @@
+const Dim_Black ="#424242";
+const GreyBlack = "#3E4059";
```

Comment:
keep the naming consistent either keep underscore as a separator or don't use it. it should be Grey_Black

Reply:
corrected